diff --git a/node_modules/@rjsf/material-ui/dist/material-ui.esm.js b/node_modules/@rjsf/material-ui/dist/material-ui.esm.js
index 27a38cf..785e58b 100644
--- a/node_modules/@rjsf/material-ui/dist/material-ui.esm.js
+++ b/node_modules/@rjsf/material-ui/dist/material-ui.esm.js
@@ -1171,41 +1171,6 @@ var ArrowUpwardIcon;
 var ErrorIcon;
 var RemoveIcon;
 
-try {
-  // core
-  Box = /*#__PURE__*/require('@material-ui/core/Box').default;
-  Button = /*#__PURE__*/require('@material-ui/core/Button').default;
-  Checkbox = /*#__PURE__*/require('@material-ui/core/Checkbox').default;
-  Divider = /*#__PURE__*/require('@material-ui/core/Divider').default;
-  FormControl = /*#__PURE__*/require('@material-ui/core/FormControl').default;
-  FormControlLabel = /*#__PURE__*/require('@material-ui/core/FormControlLabel').default;
-  FormGroup = /*#__PURE__*/require('@material-ui/core/FormGroup').default;
-  FormHelperText = /*#__PURE__*/require('@material-ui/core/FormHelperText').default;
-  FormLabel = /*#__PURE__*/require('@material-ui/core/FormLabel').default;
-  Grid = /*#__PURE__*/require('@material-ui/core/Grid').default;
-  IconButton = /*#__PURE__*/require('@material-ui/core/IconButton').default;
-  Input = /*#__PURE__*/require('@material-ui/core/Input').default;
-  InputLabel = /*#__PURE__*/require('@material-ui/core/InputLabel').default;
-  List = /*#__PURE__*/require('@material-ui/core/List').default;
-  ListItem = /*#__PURE__*/require('@material-ui/core/ListItem').default;
-  ListItemIcon = /*#__PURE__*/require('@material-ui/core/ListItemIcon').default;
-  ListItemText = /*#__PURE__*/require('@material-ui/core/ListItemText').default;
-  MenuItem = /*#__PURE__*/require('@material-ui/core/MenuItem').default;
-  Paper = /*#__PURE__*/require('@material-ui/core/Paper').default;
-  Radio = /*#__PURE__*/require('@material-ui/core/Radio').default;
-  RadioGroup = /*#__PURE__*/require('@material-ui/core/RadioGroup').default;
-  Slider = /*#__PURE__*/require('@material-ui/core/Slider').default;
-  TextField = /*#__PURE__*/require('@material-ui/core/TextField').default;
-  Typography = /*#__PURE__*/require('@material-ui/core/Typography').default; // icons
-
-  AddIcon = /*#__PURE__*/require('@material-ui/icons/Add').default;
-  ArrowDownwardIcon = /*#__PURE__*/require('@material-ui/icons/ArrowDownward').default;
-  ArrowUpwardIcon = /*#__PURE__*/require('@material-ui/icons/ArrowUpward').default;
-  ErrorIcon = /*#__PURE__*/require('@material-ui/icons/Error').default;
-  RemoveIcon = /*#__PURE__*/require('@material-ui/icons/Remove').default;
-} catch (_unused) {// no-op
-}
-
 var MaterialUIContext;
 
 if (Box && AddIcon) {
@@ -1287,74 +1252,38 @@ var Theme = /*#__PURE__*/_extends({
 
 var MuiForm = /*#__PURE__*/withTheme(Theme);
 
-/** Use require for loading these libraries in case they are not available in order to perform a useful fallback */
 // core
-
-var Box$1;
-var Button$1;
-var Checkbox$1;
-var Divider$1;
-var FormControl$1;
-var FormControlLabel$1;
-var FormGroup$1;
-var FormHelperText$1;
-var FormLabel$1;
-var Grid$1;
-var IconButton$1;
-var Input$1;
-var InputLabel$1;
-var List$1;
-var ListItem$1;
-var ListItemIcon$1;
-var ListItemText$1;
-var MenuItem$1;
-var Paper$1;
-var Radio$1;
-var RadioGroup$1;
-var Slider$1;
-var TextField$1;
-var Typography$1; // icons
-
-var AddIcon$1;
-var ArrowDownwardIcon$1;
-var ArrowUpwardIcon$1;
-var ErrorIcon$1;
-var RemoveIcon$1;
-
-try {
-  // core
-  Box$1 = /*#__PURE__*/require('@mui/material/Box').default;
-  Button$1 = /*#__PURE__*/require('@mui/material/Button').default;
-  Checkbox$1 = /*#__PURE__*/require('@mui/material/Checkbox').default;
-  Divider$1 = /*#__PURE__*/require('@mui/material/Divider').default;
-  FormControl$1 = /*#__PURE__*/require('@mui/material/FormControl').default;
-  FormControlLabel$1 = /*#__PURE__*/require('@mui/material/FormControlLabel').default;
-  FormGroup$1 = /*#__PURE__*/require('@mui/material/FormGroup').default;
-  FormHelperText$1 = /*#__PURE__*/require('@mui/material/FormHelperText').default;
-  FormLabel$1 = /*#__PURE__*/require('@mui/material/FormLabel').default;
-  Grid$1 = /*#__PURE__*/require('@mui/material/Grid').default;
-  IconButton$1 = /*#__PURE__*/require('@mui/material/IconButton').default;
-  Input$1 = /*#__PURE__*/require('@mui/material/OutlinedInput').default;
-  InputLabel$1 = /*#__PURE__*/require('@mui/material/InputLabel').default;
-  List$1 = /*#__PURE__*/require('@mui/material/List').default;
-  ListItem$1 = /*#__PURE__*/require('@mui/material/ListItem').default;
-  ListItemIcon$1 = /*#__PURE__*/require('@mui/material/ListItemIcon').default;
-  ListItemText$1 = /*#__PURE__*/require('@mui/material/ListItemText').default;
-  MenuItem$1 = /*#__PURE__*/require('@mui/material/MenuItem').default;
-  Paper$1 = /*#__PURE__*/require('@mui/material/Paper').default;
-  Radio$1 = /*#__PURE__*/require('@mui/material/Radio').default;
-  RadioGroup$1 = /*#__PURE__*/require('@mui/material/RadioGroup').default;
-  Slider$1 = /*#__PURE__*/require('@mui/material/Slider').default;
-  TextField$1 = /*#__PURE__*/require('@mui/material/TextField').default;
-  Typography$1 = /*#__PURE__*/require('@mui/material/Typography').default; // icons
-
-  AddIcon$1 = /*#__PURE__*/require('@mui/icons-material/Add').default;
-  ArrowDownwardIcon$1 = /*#__PURE__*/require('@mui/icons-material/ArrowDownward').default;
-  ArrowUpwardIcon$1 = /*#__PURE__*/require('@mui/icons-material/ArrowUpward').default;
-  ErrorIcon$1 = /*#__PURE__*/require('@mui/icons-material/Error').default;
-  RemoveIcon$1 = /*#__PURE__*/require('@mui/icons-material/Remove').default;
-} catch (_unused) {// no-op
-}
+import Box$1   from '@mui/material/Box';
+import Button$1  from '@mui/material/Button';
+import Checkbox$1  from '@mui/material/Checkbox';
+import Divider$1  from '@mui/material/Divider';
+import FormControl$1  from '@mui/material/FormControl';
+import FormControlLabel$1  from '@mui/material/FormControlLabel';
+import FormGroup$1  from '@mui/material/FormGroup';
+import FormHelperText$1  from '@mui/material/FormHelperText';
+import FormLabel$1  from '@mui/material/FormLabel';
+import Grid$1  from '@mui/material/Grid';
+import IconButton$1  from '@mui/material/IconButton';
+import Input$1  from '@mui/material/OutlinedInput';
+import InputLabel$1  from '@mui/material/InputLabel';
+import List$1  from '@mui/material/List';
+import ListItem$1  from '@mui/material/ListItem';
+import ListItemIcon$1  from '@mui/material/ListItemIcon';
+import ListItemText$1  from '@mui/material/ListItemText';
+import MenuItem$1  from '@mui/material/MenuItem';
+import Paper$1  from '@mui/material/Paper';
+import Radio$1  from '@mui/material/Radio';
+import RadioGroup$1  from '@mui/material/RadioGroup';
+import Slider$1  from '@mui/material/Slider';
+import TextField$1  from '@mui/material/TextField';
+import Typography$1  from '@mui/material/Typography'; 
+
+// icons
+import AddIcon$1 from '@mui/icons-material/Add';
+import ArrowDownwardIcon$1 from '@mui/icons-material/ArrowDownward';
+import ArrowUpwardIcon$1 from '@mui/icons-material/ArrowUpward';
+import ErrorIcon$1 from '@mui/icons-material/Error';
+import RemoveIcon$1 from '@mui/icons-material/Remove';
 
 var Mui5Context;
 
